### ISSUE TITLE
Issues with the map and loading information about polling places

### ISSUE THREAD

**Issue by feydan** [code4sac/polling-place-accessibility#1]
http://abustamam.github.io/polling-place-accessibility

**Comment by nathandcornell** [comment]
Hey @feydan! Are you able to add more details?

Honestly I'm not even sure all our data sources still exist for this.

**Comment by abustamam** [comment]
@feydan yeah this repo hasn't been actively maintained for a few years, so a lot of the public APIs probably changed somewhere down the line.

**Comment by feydan** [comment]
Sure --whenever you click on a pin the UI says "Unable to find details for your polling place."  We were looking at this project last night for one of our Hack Nights thinking it would be good to get this back up and running before this year's General Election.

Any information you can provide here about the data sources and where we could look to get this back working well would be much appreciated!  We can also add more details/issues as we investigate.

**Comment by abustamam** [comment]
@feydan thanks! Yeah at the time a lot of data from the county was stored on Google FusionTables (https://developers.google.com/apps-script/advanced/fusion-tables) which has been deprecated. I can spend some time trying to find where a new data set might be.

**Comment by costcohotdog** [comment]
How did you originally find the dataset that the SACVOTE app used? I'm pretty interested in helping get this going again for the upcoming general election.

**Comment by nathandcornell** [comment]
Try to hunt down Guy Sperry, he was the one setting up data sources for everyone during this hackathon:
https://github.com/code4sac/polling-place-accessibility#stakeholder-engagement

**Comment by costcohotdog** [comment]
Thanks Nathan!

Got in contact with Guy:
`The dataset came from inside the County. I haven't worked with them for a couple years. I know Jerry Gray is still an IT Manager there. I'll shoot him a message and see if he can get an updated copy of the source CSVs.`

**Comment by abustamam** [comment]
Thanks for following up @L0per !

**Comment by costcohotdog** [comment]
Hey All,

Does anyone happen to have a copy of the accessibility data? The county is asking because the column headers are generic:

`The accessibility data format is quite generic. Do you happen to have a copy of that data set downloaded that you can share?  We’re looking through our archives, but have not been able to find a copy.   I want to try to get the equivalent info from the new system if possible.`

**Comment by abustamam** [comment]
Unfortunately I don't, but I believe some of the requests have some references to some headers, such as here:

https://github.com/code4sac/polling-place-accessibility/blob/master/src/actions/stories/elevators.js

```js
var elevatorsData = pollingPlace['Elevators & Lifts'].root;
var outsideElevator = pollingPlace['Elevators & Lifts']['1-Main-Entry-Floor-Outside-the-Elevator'];
var votingArea = pollingPlace['Elevators & Lifts']['2-Voting-Area-Floor-Outside-the-Elevator'];
var inside = pollingPlace['Elevators & Lifts']['3-Inside-the-Elevator'];
var controls = pollingPlace['Elevators & Lifts']['4-Control-Panel'];
var emergency = pollingPlace['Elevators & Lifts']['5-Emergency-Controls'];
var carsize = pollingPlace['Elevators & Lifts']['6-Car-Dimensions'];
var wheelchair = pollingPlace['Elevators & Lifts']['8-Wheelchair-Lifts'];
```

Hope this is helpful.
